cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/runtimejob/DataprocRuntimeJobManager.java

```diff
@@ -201,6 +202,9 @@ public void launch(RuntimeJobInfo runtimeJobInfo) throws Exception {
       LOG.debug("Successfully submitted hadoop job {} to cluster {}.", job.getReference().getJobId(), clusterName);
       DataprocUtils.emitMetric(provisionerContext, region,
                                "provisioner.submitJob.response.count");
+    } catch (AlreadyExistsException ex) {
+      //the job id already exists, ignore the job.
+      LOG.warn("The dataproc job {} already exists. Ignoring resubmission of the job.", ex.getMessage());
```

Comment:
log parameter does not seem to be job name.

Reply:
good catch, done :)